client/src/pages/readme.md

```diff
@@ -0,0 +1,38 @@
+# YOCO
+
+This project is called "Donation Form", and it's a React application that allows users to make a donation to the Hope Again Recovery Home.
```

Comment:
The wording here is unclear, you can mention that you are using yoco for donations